examples/webgpu_postprocessing_ssr.html

```diff
@@ -0,0 +1,232 @@
+
+			scene = new THREE.Scene();
+			scene.background = new THREE.Color( 0x443333 );
+			scene.fog = new THREE.Fog( 0x443333, 1, 4 );
+
+			// Ground
+			const plane = new THREE.Mesh(
+				new THREE.PlaneGeometry( 8, 8 ),
+				new THREE.MeshPhongMaterial( { color: 0xcbcbcb } )
+			);
+			plane.rotation.x = - Math.PI / 2;
+			plane.position.y = - 0.0001;
+			scene.add( plane );
+
+			// Lights
+			const hemiLight = new THREE.HemisphereLight( 0x8d7c7c, 0x494966, 3 );
+			scene.add( hemiLight );
+
+			const spotLight = new THREE.SpotLight();
+			spotLight.intensity = 8;
+			spotLight.angle = Math.PI / 16;
+			spotLight.penumbra = 0.5;
+			spotLight.position.set( - 1, 1, 1 );
+			scene.add( spotLight );
+
+			dracoLoader.load( 'models/draco/bunny.drc', function ( geometry ) {
+
+				geometry.computeVertexNormals();
+
+				const material = new THREE.MeshStandardMaterial( { color: 0xa5a5a5, metalness: 0.01 } );
+				const mesh = new THREE.Mesh( geometry, material );
+				mesh.position.y = - 0.0365;
+				scene.add( mesh );
+
+				dracoLoader.dispose();
+
+			} );
+
+			let geometry, material, mesh;
+
+			geometry = new THREE.BoxGeometry( .05, .05, .05 );
+			material = new THREE.MeshStandardMaterial( { color: 'green', metalness: 0.01 } );
+			mesh = new THREE.Mesh( geometry, material );
+			mesh.position.set( - .12, .025, .015 );
+			scene.add( mesh );
+
+			geometry = new THREE.IcosahedronGeometry( .025, 4 );
+			material = new THREE.MeshStandardMaterial( { color: 'cyan', metalness: 0.01 } );
+			mesh = new THREE.Mesh( geometry, material );
+			mesh.position.set( - .05, .025, .08 );
+			scene.add( mesh );
+		
+			geometry = new THREE.ConeGeometry( .025, .05, 64 );
+			material = new THREE.MeshStandardMaterial( { color: 'yellow', metalness: 0.01 } );
+			mesh = new THREE.Mesh( geometry, material );
+			mesh.position.set( - .05, .025, - .055 );
+			scene.add( mesh );
+
+			//
+
+			renderer = new THREE.WebGPURenderer();
+			renderer.setPixelRatio( window.devicePixelRatio );
+			renderer.setSize( window.innerWidth, window.innerHeight );
+			renderer.setAnimationLoop( animate );
+			document.body.appendChild( renderer.domElement );
+
+			//
+
+			postProcessing = new THREE.PostProcessing( renderer );
+
+			const scenePass = pass( scene, camera, { minFilter: THREE.NearestFilter, magFilter: THREE.NearestFilter } );
+			scenePass.setMRT( mrt( {
+				output: output,
+				normal: transformedNormalView,
+				metalness: metalness
+			} ) );
+
+			const scenePassColor = scenePass.getTextureNode( 'output' );
+			const scenePassNormal = scenePass.getTextureNode( 'normal' );
+			const scenePassDepth = scenePass.getTextureNode( 'depth' );
+			const scenePassMetalness = scenePass.getTextureNode( 'metalness' );
+
+			ssrPass = ssr( scenePassColor, scenePassDepth, scenePassNormal, scenePassMetalness, camera );
+
+			// blend SSR over beauty
+		
+			const outputNode = vec4( scenePass.rgb.mul( ssrPass.a.oneMinus() ).add( ssrPass.rgb.mul( ssrPass.a ) ), scenePass.a );
```

Comment:
@sunag This imitates the previous normal blending but I hope we can find an easier way to blend to reflective colors onto the beauty pass.